src/lib.rs

```diff
@@ -25,6 +26,8 @@ pub use crate::{
     version::{version, LibraryVersion},
 };
 
+pub use libusb1_sys::constants::LIBUSB_CONTROL_SETUP_SIZE as CONTROL_SETUP_SIZE;
```

Comment:
Why you can use `rusb::constants::LIBUSB_CONTROL_SETUP_SIZE`?

Reply:
I didn't realize that it was already exported like that.